src/mpyl/steps/test/dockertest.py

```diff
@@ -85,15 +86,14 @@ def execute(self, step_input: Input) -> Output:
                 message=f"Tests results produced for {project.name} ({summary})",
                 produced_artifact=artifact,
             )
+            remove_container(self._logger, container)
```

Comment:
Should we make `remove_container` fail safe perhaps? Or is there no way in which a container could have been created when `success == False`?

Reply:
The container only gets created inside the `if success` now, that's why I moved the remove as well.

This pr isn't done yet btw, the above changes helped with the logging/debugging but didn't fix the test step of the payment service yet. I have a meeting today with Rogier to figure out what's going on with it.

Reply:
In that case, don't forget to replace `COPY . .` with `COPY payment/ payment/` :-)